deenurp/subcommands/gb2csv.py

```diff
@@ -0,0 +1,81 @@
+"""
+Convert a genbank record to csv format
+
+Output:
+
+Genbank
+[id, name, description, version, accession, gi, tax_id,
+ comment, date, source, keywords, organism]
+
+References
+[version, title, authors, comment,
+ consrtm, journal, medline_id, pubmed_id]
+"""
+
+import argparse
+import csv
+import re
+import sys
+
+from Bio import SeqIO
+
+from deenurp.util import accession_version_of_genbank, tax_of_genbank, Counter
+
+
+def build_parser(parser):
+    parser.add_argument('infile',
+                        default=sys.stdin,
+                        nargs='?',
+                        help="""path to genbank file [default: stdin]""")
+    parser.add_argument('--references-out',
+                        type=argparse.FileType('w'),
+                        help="""output references""")
+    parser.add_argument('--out',
+                        type=argparse.FileType('w'),
+                        default=sys.stdout,
+                        help="""output path [default: stdout]""")
+
+
+def action(args):
+    records = SeqIO.parse(args.infile, format='gb')
+    records = Counter(records, prefix='Record ')
+
+    fieldnames = ['version', 'accession', 'id', 'name', 'description',
```

Comment:
Can we have a command line option to define fieldnames, and list these in the docstring?